go.mod

```diff
@@ -37,6 +39,8 @@ require (
 	k8s.io/api v0.23.3
 	k8s.io/apimachinery v0.23.3
 	k8s.io/client-go v0.23.3
+	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
+	sigs.k8s.io/cli-utils v0.27.0
```

Comment:
```suggestion
	sigs.k8s.io/cli-utils v0.28.0
```

Reply:
Package version has been changed, and copyright headers have been updated for at least the `api/v1beta2` package. If there are remaining ones, I will address those in `rewrite-spec` (which also touches some in-code comments).